data/database.alter.sql

```diff
@@ -2176,3 +2176,10 @@ ALTER TABLE schedules DROP COLUMN staff;
 
 
 ALTER TABLE finna_additions ADD COLUMN custom_data jsonb;
+
+
+ALTER TABLE organisations ADD library_system_name varchar(255);
+COMMENT ON COLUMN organisations.library_system_name IS 'Machine name of the system in use (sierra, axiell, koha)';
+
+ALTER TABLE organisations ADD library_system_id varchar(255);
```

Comment:
After our discussion about what this represents my thought here is that maybe it would be better to call this field "library_system_servicepoint_id". Would that be an appropriate name and are there any issues changing it? The benefit would be clear at first look that it doesn't mean a library system (instance or system type) specific identifier?

The comment on the column could maybe be a little more specific about the purpose too.

Reply:
Should renaming `library_system_id` to `library_system_servicepoint_id` reflect to everything else, like for example the kirkanta api (query param names, json payload, etc)?

Reply:
Yeah I think that would be ideal if it would since it cuts through all those as the same thing, does doing that cause complications?

Reply:
No, not really. I will implement the changes now.